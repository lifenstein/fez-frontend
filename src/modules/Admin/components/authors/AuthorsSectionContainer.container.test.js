import { AuthorsSectionContainer } from './AuthorsSectionContainer';

function setup(testProps = {}, args = { isShallow: true }) {
    const props = {
        formValues: {
            toJS: jest.fn(() => ({})),
        },
        ...testProps,
    };

    return getElement(AuthorsSectionContainer, props, args);
}

describe('AuthorsSectionContainer component', () => {
    it('should render default view', () => {
        const wrapper = setup();
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render disabled view', () => {
        const wrapper = setup({ disabled: true });
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
