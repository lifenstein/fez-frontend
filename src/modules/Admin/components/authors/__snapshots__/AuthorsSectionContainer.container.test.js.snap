// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorsSectionContainer component should render default view 1`] = `
<ContextProvider
  value={
    Object {
      "formValues": Object {
        "isNtro": false,
      },
    }
  }
>
  <Memo(AuthorsSection)
    isNtro={false}
    key="disabled-undefined-isNtro-false"
  />
</ContextProvider>
`;

exports[`AuthorsSectionContainer component should render disabled view 1`] = `
<ContextProvider
  value={
    Object {
      "formValues": Object {
        "isNtro": false,
      },
    }
  }
>
  <Memo(AuthorsSection)
    disabled={true}
    isNtro={false}
    key="disabled-true-isNtro-false"
  />
</ContextProvider>
`;
